packages/hub/src/components/layout/RootLayout/UserControls.tsx

```diff
@@ -52,6 +76,27 @@ export const UserControls: FC<{ session: Session | null }> = ({ session }) => {
               icon={SignOutIcon}
               title="Sign Out"
             />
+            <DropdownMenuSeparator />
+            <DropdownMenuHeader>Resources</DropdownMenuHeader>
```

Comment:
These would make more sense to me somewhere else.

Suggestion: rename `Docs` menu item to `Resources`, convert it to dropdown with 4 items (`Squiggle docs` first and three other links below).

Reply:
Yea, I wasn't sure here.

Github has one big dropdown with everything, so I was taking inspiration from that.

"Docs" as it is is much more important than the rest of these, so I want to give it extra attention.

Will consider in the future.